src/os/bluestore/BlueStore.cc

```diff
@@ -7765,6 +7766,31 @@ void BlueStore::_fsck_check_object_omap(FSCKDepth depth,
         << fsck_dendl;
     }
   }
+  if (!o->onode.is_perpool_omap() && !o->onode.is_pgmeta_omap()) {
```

Comment:
This would produce duplicate warning/error for onode in a format prior to "perpool".

wouldn't it be better to rearrange this and above 'if's in the following way:
if (!o->onode.is_perpool_omap() && !o->onode.is_pgmeta_omap()){
...
} else if (!o->onode.is_perpg_omap() && !o->onode.is_pgmeta_omap()){
...
}